README.md

```diff
@@ -9,6 +9,129 @@ Firebase Auth with Rocket, batteries included
 - **Tiny**: `rocket-firebase-auth` is tiny, with features allowing you to make it even tinier
 - **Does one thing well**: Encodes/decodes Firebase JWT tokens in Rocket apps, and that's it
 
+## Quickstart:
```

Comment:
`## QuickStart` seems to be the same as `## Getting started`. Could I ask why you added this?

Reply:
Yes, they're basically the same thing.

The reason I added a "Quickstart" section at the beginning because I almost clicked off this crate when I was looking for a firebase auth system one because there wasn't a quick and intuitive example on how to use it right as I opened it up. Remember, people are looking for a quick solution when browsing crates.

It's kind of a marketing thing, new users might get discouraged from using it when they first have to scroll through an update section and then read a three part "getting started" guide when an example codeblock tells you all you need.

However I do agree that it's dumb to have the same thing in two places of the README.

Reply:
Yeah, having the upgrade guide as the first section of the Readme is a bit confusing. I'll place it lower down on the page 👍